src/components/Heading/Heading.tsx

```diff
@@ -110,5 +110,3 @@ export const Heading = forwardRef(
 );
 
 Heading.displayName = 'Heading'; // Satisfy eslint.
-
-export default Heading;
```

Comment:
Are we generally expecting to not use default exports for components in EDS now?

Reply:
The index file for the respective component exports these as default already, but we don't need default exporting (and haven't been) for the component file itself, i.e. `import Heading from '@czi/eds/lib/components/Heading` works and now removing `import Heading from @czi/eds/liib/components/Heading/Heading.tsx`. These are being still being re-exported in `src/index` anyways so general use is still `import {Heading} from @czi/eds`.